pythonFiles/vscode_datascience_helpers/getVariableInfo/vscodeGetVariableInfo.py

```diff
@@ -1,73 +1,90 @@
-# Query Jupyter server for the info about a dataframe
-import json as _VSCODE_json
-import builtins as _VSCODE_builtins
+def _VSCODE_getVariable(what_to_get, *args):
```

Comment:
@rchiodo I've modified the code so that everything is under a function now, all we need to do is delete the imported modules & delete this function that we declare (when cleaning up).
If we were to import a module, we'd still need to delete the imported module, 

Now we should not end up with tonnes of function definitinos as everything is local to this one function, hence no need to clean those individual functions (as its all local & goes out of scope)